src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/EditorConfigFiles.xaml

```diff
@@ -67,6 +67,14 @@
     </StringProperty.Metadata>
   </StringProperty>
 
+  <StringProperty Name="NuGetPackageId"
+                  Visible="false">
+    <StringProperty.Metadata>
+      <NameValuePair Name="DoNotCopyAcrossProjects"
+                     Value="true" />
+    </StringProperty.Metadata>
+  </StringProperty>
+  
```

Comment:
A little surprised to see this for editor config files.

Reply:
NuGet can use any item, we have a specific item for EditorConfigFiles for reasons I don't know, so we need to include it. All other items not listed are going to use the None rule.